handlers/feedback.go

```diff
@@ -152,25 +152,35 @@ func addFeedback(w http.ResponseWriter, req *http.Request, isPositive bool, rend
 		return
 	}
 
-	if len(f.Email) > 0 && !isPositive {
-		if ok, err := regexp.MatchString(`^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,6}$`, f.Email); !ok || err != nil {
-			getFeedback(w, req, f.URL, "email", f.Description, f.Name, f.Email, lang, rend, cacheService)
-			return
-		}
+	var isGeneralFeedback bool
```

Comment:
We should find out where this bool value should be obtained from.

Reply:
Not used. Was removed.